Cargo.toml

```diff
@@ -1,19 +1,29 @@
 [package]
-name = "landlock"
+name = "landlock-sandboxer"
 version = "0.1.0"
 authors = [
 	"Mickaël Salaün <mic@digikod.net>",
 	"Vincent Dagonneau <vincentdagonneau@gmail.com>"
 ]
 edition = "2018"
-description = "Landlock LSM helpers"
+description = "Landlock LSM sandboxer"
 homepage = "https://landlock.io"
 repository = "https://github.com/landlock-lsm/rust-landlock"
 license = "MIT OR Apache-2.0"
 keywords = ["access-control", "linux", "sandbox", "security"]
 exclude = [".gitignore"]
 readme = "README.md"
 
+[[bin]]
+bench = false
+path = "crates/sandboxer/main.rs"
```

Comment:
Let's simply put it in `examples/sandboxer.rs` for now and not move all other files. It could then be run with `cargo run --example=sandboxer`.
We can also add the required dependencies with `[dev-dependencies]`.